data/json/effects.json

```diff
@@ -1514,5 +1514,53 @@
             "speed_mod": [-1000],
             "dex_mod": [-100]
         }
+    },
+    {
+        "type": "effect_type",
+        "id": "brittlebone",
+        "name": [ "Brittle bones" ],
+        "desc": [ "A lack of calcium in your diet has made your bones brittle" ],
+        "apply_message": "Your bones have become brittle",
+        "rating": "bad"
+    },
+    {
+        "type": "effect_type",
+        "id": "anaemia",
+        "name": [ "Anaemia" ],
+        "desc": [ "A lack of iron in your diet has made you anaemic" ],
+        "apply_message": "You feel tired and listless",
+        "rating": "bad"
+    },
+    {
+        "type": "effect_type",
+        "id": "nightblind",
+        "name": [ "Night blindness" ],
+        "desc": [ "A lack of vitamin A in your diet has affected your vision" ],
+        "apply_message": "You struggle to make out the finer details",
+        "rating": "bad"
+    },
+    {
+        "type": "effect_type",
+        "id": "slowheal",
+        "name": [ "Slow healing" ],
+        "desc": [ "A lack of vitamin B12 in your diet has affected your ability to heal" ],
+        "apply_message": "Simple wounds concern you more than usual",
+        "rating": "bad"
+    },
+    {
+        "type": "effect_type",
+        "id": "scurvy",
+        "name": [ "Scurvy" ],
+        "desc": [ "A lack of vitamin C in your diet has resulted in you developing scurvy" ],
```

Comment:
Ah right I see. Scurvy was going to be low level (10-20) pain with a recurrent chance for infection (to simulate bleeding sores). Tying that in to deficiency level could work especially since each vitamin specifies a lower bound